srcdeps-maven-quickstarts/srcdeps-mvn-git-revision-non-master-quickstart/.mvn/extensions.xml

```diff
@@ -23,4 +23,9 @@
     <artifactId>srcdeps-maven-local-repository</artifactId>
     <version>2.0.2</version><!-- @srcdeps.version@ -->
   </extension>
+  <extension>
+    <groupId>org.srcdeps.mvn</groupId>
+    <artifactId>srcdeps-maven-enforcer</artifactId>
+    <version>2.0.2</version><!-- @srcdeps.version@ -->
```

Comment:
For what the users see when they browse here on Github or when they clone and experiment locally, the version has to be the last stable - that's the didactic function of these quickstarts. They need to work without compiling srcdeps and they should show the best practices.

OTOH, when running these quickstarts a itests during the build of srcdeps, these versions need to be the current -SNAPSHOT. We replace them dynmically when copying the test projects https://github.com/ppalaga/srcdeps-maven/blob/53df401de1a151bace02f019c8018ffd60535578/srcdeps-maven-local-repository/src/test/java/org/srcdeps/mvn/localrepo/SrcdepsLocalRepositoryManagerTest.java#L80 that's why the `<!-- @srcdeps.version@ -->` marker is there.